pkg/eosclient/eosgrpc/eosgrpc.go

```diff
@@ -181,9 +180,28 @@ func New(opt *Options) *Client {
 	return c
 }
 
+// If the error is not nil, take that
+// If there is an error coming from EOS, erturn a descriptive error
+func (c *Client) getRespError(rsp *erpc.NSResponse, err error) error {
+	if err != nil {
+		return err
+	}
+
+	if rsp.Error.Code == 0 {
+		return nil
+	}
+
+	err2 := errtypes.InternalError("Err from EOS: " + fmt.Sprintf("%#v", rsp.Error))
+	return err2
+}
+
 // Common code to create and initialize a NSRequest
-func (c *Client) initNSRequest(uid, gid string) (*erpc.NSRequest, error) {
+func (c *Client) initNSRequest(ctx context.Context, uid, gid string) (*erpc.NSRequest, error) {
 	// Stuff filename, uid, gid into the MDRequest type
+
+	log := appctx.GetLogger(ctx)
+	log.Debug().Str("(uid,gid)", "("+uid+","+gid+")").Msg("New grpcNS req")
```

Comment:
`log.Debug().Str("uid", uid).Str("gid", gid).Msg("New grpcNS req")` could be a cleaner alternative

Reply:
Not sure it's cleaner, at least for human reading. I did that way because this logger component does not even guarantee the ordering of the items it logs. Actually I find it a pretty bad logging component also for other reasons, yet...